src/App.tsx

```diff
@@ -0,0 +1,20 @@
+import SidebarLeft from "./components/SidebarLeft";
+import SidebarRight from "./components/SidebarRight";
+import Main from "./components/Main";
+import "./assets/styles/global.css";
+import "./assets/styles/main.css";
+import "./assets/styles/sidebar-left.css";
```

Comment:
why did you import all styles here and not on each main component, I mean
import "./assets/styles/sidebar-left.css" inside SidebarLeft and same for the right side bar

Reply:
I found that importing all the styles in the main App component helped me keep my styles centralized, that way I am sure the classes are applied to the main component and then passed to each individual component

Reply:
Alright but importing everything here can produce conflicts between components?

Reply:
Yes, it's prone to naming conflicts and style duplications, with my current approach I have to be very careful with class naming, to avoid this I can use other stying options like CSS-in-JS or CSS modules